.azure-pipelines/azure-pipelines-job-as-stage.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
```

Comment:
it is unclear why these changes are related to the pr.

Reply:
The official build is using the file, supports to build single platform for platform while using the same yaml file.
The old one build-template.yml is no use any more, can be removed.

Reply:
@lguohan , I may not understand your concern, could you please give more detail about it? We can talk about it if needed.

Reply:
according to the description, it increases the timeout from 600m to 720m. then, isn't line 22 on the job-groups.yml doing that? why do we need to add this stage.yml?

Reply:
Oh, I got it. I thought only one line change in my PR. The stage.yml was for my test only, it got merged by mistake. I have removed it. Thanks.